sbpy/photometry/iau.py

```diff
@@ -7,19 +7,22 @@
 
 """
 
+
+__all__ = ['HG', 'HG12', 'HG1G2', 'HG12_Pen16']
```

Comment:
I see you moved the `__all__` line to the beginning of this file.  In many other files, it's after the imports.  This is only a style issue and doesn't really matter, but should we keep a consistency internally?

Reply:
PEP8 recommends "dunders" like __all__ before imports: https://peps.python.org/pep-0008/#module-level-dunder-names   We generally should try to follow PEP8, but it's mostly just helpful, not required.